src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -30,19 +30,57 @@ class InstallLibertyTask extends AbstractTask {
     }
 
     private Map<String, String> buildInstallLibertyMap(Project project) {
+
         Map<String, String> result = new HashMap();
         if (project.liberty.install.licenseCode != null) {
            result.put('licenseCode', project.liberty.install.licenseCode)
         }
 
         if (project.liberty.install.version != null) {
             result.put('version', project.liberty.install.version)
+            version = project.liberty.install.version
+        }
+        
+        if (project.liberty.install.type != null) {
+            result.put('type', project.liberty.install.type)
+            type = project.liberty.install.type
         }
 
         if (project.liberty.install.runtimeUrl != null) {
             result.put('runtimeUrl', project.liberty.install.runtimeUrl)
-        }
+        } 
+        else if (project.liberty.assemblyArtifact.groupId != null) {
```

Comment:
I am still confused with the mix and match of the `install` and the `assemblyArtifact` blocks.  I would expect these to be either/or.